fv3config/_datastore.py

```diff
@@ -187,6 +207,35 @@ def get_field_table_filename(config):
     Returns:
         str: field_table filename
 
+    Raises:
+        ConfigError
+    """
+    field_table = config.get("field_table", None)
+    if field_table is None:
+        directory = os.path.join(DATA_DIR, "field_table")
+        return get_inferred_field_table_filename(config, directory)
+    else:
+        if not filesystem.is_existing_absolute_path(field_table):
+            raise ConfigError(
+                f"field_table={field_table} must either be left unset or set "
+                "to an existing absolute path to a file or directory"
+            )
```

Comment:
We talked on the phone briefly about combining the three checks of this into a function call that occurs only once at the end on a field_table path. You will also have to check if the given `field_table` is a directory, but it won't be necessary to raise if it doesn't exist because the file will also not exist in the check at the end.

Reply:
Thanks @mcgibbon!  I think I figured out a decent solution.  Please have another quick look.